### PR TITLE
Sortkeys

### PR DESCRIPTION
**Pull request by Decodetalkers**
resolve: https://github.com/linuxdeepin/developer-center/issues/5117

### CODE REVIEW

**Review by dengbo11 — APPROVED** [review]

File: tools/override/main.go [hunk 1]
```diff
@@ -151,8 +165,10 @@ func combineFiles(inputFiles []string, outputFile string) (err error) {
 			return
 		}
 		sections := kf.GetSections()
+
 		for _, section := range sections {
```
Comment (black-desk) [inline tools/override/main.go:169]: Should we make `sections` sorted too?

**Review by black-desk — CHANGES_REQUESTED** [review]

File: tools/override/main.go (hunk 1 above)
Comment (Decodetalkers) [reply tools/override/main.go:169]: I think we need

File: tools/override/main.go [hunk 2]
```diff
@@ -139,6 +140,28 @@ func getValue(value string, kf *keyfile.KeyFile, section0 string) string {
 	return value
 }
 
+func mapToSortedList(basket map[string]string) []struct {
+	key string
```
Comment (black-desk) [inline tools/override/main.go:144]: Should we define this {key,val} struct some where so that we can void keep define it inline?

**Review by black-desk — CHANGES_REQUESTED** [review]

File: tools/override/main.go (hunk 2 above)
Comment (Decodetalkers) [reply tools/override/main.go:144]: ok, changed

**Review by black-desk — APPROVED** [review]

**Review by justforlxz — APPROVED** [review]

**Pull request merged**